src/main/java/dev/openfeature/sdk/OpenFeatureClient.java

```diff
@@ -1,24 +1,16 @@
 package dev.openfeature.sdk;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.List;
-import java.util.function.Consumer;
-
-import dev.openfeature.sdk.exceptions.ExceptionUtils;
-import dev.openfeature.sdk.exceptions.FatalError;
-import dev.openfeature.sdk.exceptions.GeneralError;
-import dev.openfeature.sdk.exceptions.OpenFeatureError;
-import dev.openfeature.sdk.exceptions.ProviderNotReadyError;
+import dev.openfeature.sdk.exceptions.*;
```

Comment:
please no star imports

Reply:
Thanks! I think the IDE "optimized" this. I need to figure out how to configure IntelliJ to avoid wildcard imports for regular files while still allowing them for test files. This is probably still useful for things like static imports of Mockito assertions in tests...

Reply:
i started with https://github.com/open-feature/java-sdk/pull/1098 i should finish it, to prevent this in the future